src/foundry/client/pixi/core/loader.d.ts

```diff
@@ -43,62 +47,55 @@ declare global {
     loadTexture(src: string): Promise<PIXI.BaseTexture>;
 
     /**
-     * Log texture loading progress in the console and in the Scene loading bar
-     * @internal
-     */
-    protected _onProgress(src: string, progress: TextureLoader.Progress): void;
-
-    /**
-     * Log failed texture loading
-     * @internal
```

Comment:
Ah this is another rule we should write down: We do keep `@internal` if it's in the source, but if the source is marked `@private`, that is translated to `@internal` in the types. (`@private` is not a valid TSDoc tag while valid in JSDoc) So only remove these if they were removed in the source.

Reply:
Updated

Reply:
These did get reordered and renamed in the v10 update. So now that are `#onError`, etc., nearer to the bottom.